### ISSUE TITLE
Error during VM creation on opennebula 5.2

### ISSUE THREAD

**Issue by stewpan** [OpenNebula/terraform-provider-opennebula#112]
Sometimes and sometimes not I got an error during VM creation with the following:

```
402 Error waiting for virtual machine (97851) to be in state RUNNING: GOCA client error [REQUEST_HTTP]: http make request: Post "https://api.hpccloud.surfsara.nl/RPC2": EOF
``` 

Not sure how to solve  this or what exactly causes it, maybe because my provider uses such an old version of opennebula?

**Comment by stewpan** [comment]
Not sure why, but when I edit the code to wait for a running state it runs without problem after line 398.
```diff
+      //Ensure the state is Running
+	vm, err := vmc.Info(false)
+	for vmState, vmLcmState, _ := vm.State(); vmLcmState.String() == "RUNNING"; _, vmLcmState, _ = vm.State() {
+		fmt.Printf("Sharing info %+v , ... ., %v\n", vmState, vmLcmState)
+	}

```

**Comment by jaypif** [comment]
Hi @Stvdputten 

Unfortunately the Terraform provider has been developed and tested only for OpenNebula release starting from 5.6.

The goca (OpenNebula API wrapper) has been officialy released with 5.6.

The behavior before 5.6 cannot be guaranteed.

If possible I invite you to test on a recent release.

Regards

**Comment by stewpan** [comment]
Thank you for the response, @jaypif ! 

I fear I'm stuck with 5.2! I'll use my current solution, which is fine for now.
